integration_test/pg/test_helper.exs

```diff
@@ -79,11 +79,18 @@ version =
     _other -> version
   end
 
-if Version.match?(version, ">= 9.5.0") do
-  ExUnit.configure(exclude: [:without_conflict_target])
-else
-  Application.put_env(:ecto_sql, :postgres_map_type, "json")
-  ExUnit.configure(exclude: [:upsert, :upsert_all, :array_type, :aggregate_filters])
+excludes_above_9_5 = [:without_conflict_target]
+excludes_below_9_5 = [:upsert, :upsert_all, :array_type, :aggregate_filters]
+excludes_below_9_6 = [:add_column_if_not_exists, :no_error_on_add_column_migration]
```

Comment:
We should use the same tag for both of those tests but since I commented above that we should merge those tests, then it is just a matter of remembering to update this too.